### DISCUSSION TITLE
Feature Request: Automatic Cleanup Button

### DISCUSSION THREAD

**Discussion by ghost** [jgraph/drawio#3867] [General]
Hi, I had an idea for a feature that I think would be really useful. Essentially, it would be a button you could press that would clean up your graph. I know that's a really bad way to say it and doesn't explain anything, so I'll just throw some images in to show what I'm talking about.
![1](https://github.com/jgraph/drawio/assets/139376564/308d6a70-5664-4347-b90e-ff569da5b759)

![2](https://github.com/jgraph/drawio/assets/139376564/46a062f1-986d-4978-a4d5-83666af5cd8a)

The pictures should hopefully explain everything. I know this would probably be a lot of work to add, but I figured there was no harm in just suggesting it. Thanks!

**Comment by alderg** [comment]
Try Arrange, Layout, Organic.

**Reply by ghost** [reply]
Thank you!
